.github/workflows/release.yaml

```diff
@@ -0,0 +1,25 @@
+name: Generate release assets
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  releases-matrix:
+    name: Release precompiled binaries
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goos: [linux]
+        goarch: [amd64, arm64]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: wangyoucao577/go-release-action@v1
```

Comment:
can you please use a hash here? People can change the tag and let it point to something bad

Reply:
Done